src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -407,6 +413,105 @@ public Response updateDraftVersion( String jsonBody, @PathParam("id") String id,
         }
     }
     
+    @PUT
+    @Path("{id}/addData")
+    public Response addDataToVersion(String jsonBody, @PathParam("id") String id) throws WrappedResponse{
+
+        DataverseRequest req = createDataverseRequest(findUserOrDie());
+
+        return processDatasetUpdate(jsonBody, id, req, true);
+    }
+    
+    @PUT
+    @Path("{id}/updateData")
+    public Response updateDataInVersion(String jsonBody, @PathParam("id") String id)  throws WrappedResponse {
```

Comment:
Both endpoints do the exact same thing? A usability decision?

Reply:
mistake as noted below